CMakeLists.txt

```diff
@@ -330,6 +330,17 @@ if (TBB_EXAMPLES)
     add_subdirectory(examples)
 endif()
 
+
+if (TBB_DOC_EXAMPLES)
+    set(TBB_DOC_EXAMPLES_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/doc/main/examples_testing")
+    if (NOT EXISTS ${TBB_DOC_EXAMPLES_DIRECTORY})
+        message(FATAL_ERROR "Documentation examples are not found while testing was enabled")
```

Comment:
Do we really want to fail here? Maybe the warning will be enough?

Reply:
The idea actually was to prevent some of our automatic workloads to silently fail if the `examples_testing` directory will be missed of moved somewhere. I believe in that case `ctest` will just run nothing and return the successful status, but no testing would be done.